exercises/1.3/ros2/src/myworkcell_core/CMakeLists.txt

```diff
@@ -23,12 +23,9 @@ find_package(rclcpp REQUIRED)
 add_executable(vision_node src/vision_node.cpp)
 ament_target_dependencies(vision_node rclcpp)
 
-# Mark executables and/or libraries for installation
+# Mark executables for installation
 install(TARGETS vision_node
-    ARCHIVE DESTINATION lib/${PROJECT_NAME}
-    LIBRARY DESTINATION lib/${PROJECT_NAME}
-    RUNTIME DESTINATION lib/${PROJECT_NAME}
-)
+    DESTINATION lib/${PROJECT_NAME})
```

Comment:
Might still be good to leave the `RUNTIME` flag so people don't end up using the same command for libraries (which should go in `lib/`)

Reply:
I was just sticking with the format that is used when running `ros2 pkg create  --node-name my_node my_package`.    I'll plan to just leave it as-is for now, but I see your point.

Reply:
I added a comment in the wiki text to clarify this difference.